Makefile

```diff
@@ -1,29 +0,0 @@
-# Set the task name
-TASK = taco
```

Comment:
Might be easiest to just put the Makefile back.   And then there should be some decision on if we want to keep this in $SKA/share/taco or move to $SKA/share/acis_taco.   I'm specifically concerned about the make_esaview_data.py cron job and the esaview.py script.  Thanks!